server/server.go

```diff
@@ -383,16 +411,16 @@ func configureStorage(logger *zerolog.Logger, conf *Config) (storageProvider sto
 		}
 	}
 
-	if conf.ChainID == flowgo.Testnet || conf.ChainID == flowgo.Mainnet {
+	if conf.ForkHost != "" {
```

Comment:
Should we add some kind of validation to ForkHost string like missing port, url scheme etc? it will fail regardless, but at least in this way we can notify the user with a proper error message